tests/conftest.py

```diff
@@ -1451,9 +1451,9 @@ def finalizer():
 
 
 @pytest.fixture()
-def dc_pod_factory(request, pvc_factory, service_account_factory):
+def deploy_pod_factory(request, pvc_factory, service_account_factory):
```

Comment:
more intuitive to rename the function to `deployment_pod_factory`

Reply:
Resolved with latest commit